recipe/meta.yaml

```diff
@@ -19,8 +21,8 @@ requirements:
     - pkg-config  # [unix]
   host:
     - python
+    - poetry >=0.12
```

Comment:
Could you comment why this is needed? Moreover, it seems to cause version conflicts.

Reply:
[poetry is a build dependency here](https://github.com/matze/pkgconfig/blob/master/pyproject.toml#L2). That is why the previous one failed in all build matrix items. Also, poetry is not migrated to py39 yet nor osxarm64, so that is why we cannot build this version there. I recommend just merging this if you need this version or wait until we have poetry in those platforms and Python versions